### DISCUSSION TITLE
problem with deploy when doing push

### DISCUSSION THREAD

**Discussion by projettajr** [squidfunk/mkdocs-material#3133] [Q&A] [answered]
Hi, I'm quite a layman on this issue, but I have a documentation page for the junior company I'm part of, and we were using it perfectly, to pass information between members and tutorials about the company's processes, but from yesterday to today the deploys appear with this error. Any idea how to solve it, I don't have in-depth knowledge.

![image](https://user-images.githubusercontent.com/79222564/138577926-1e74b1eb-90a4-4991-b66e-bf7cfa9bc1e1.png)

**Comment by facelessuser** [comment, marked as answer]
Hmm, maybe a MkDocs bug. Maybe they meant to import `Sequence` from `collections.abc` instead of `collections`?

**Reply by squidfunk** [reply]
Maybe Python 3.10? Is that new?

**Reply by facelessuser** [reply]
Yeah, it's been deprecated for a while:

```py
>>> from collections import Sequence
<stdin>:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
```

**Reply by squidfunk** [reply]
Thanks for testing. Thus, it's indeed a MkDocs bug that should be reported [upstream](https://github.com/mkdocs/mkdocs) 😊
